modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMDataDefinitionConverterImpl.java

```diff
@@ -80,6 +111,26 @@ private DDMFormFieldOptions _getDDMFormFieldOptions(
 		return ddmFormFieldOptions;
 	}
 
+	private String _getDDMFormFieldsRows(DDMFormField fieldSetDDMFormField) {
+		JSONArray jsonArray = JSONFactoryUtil.createJSONArray();
+
+		for (DDMFormField ddmFormField :
+				fieldSetDDMFormField.getNestedDDMFormFields()) {
+
+			jsonArray.put(
+				JSONUtil.put(
+					"columns",
+					JSONUtil.putAll(
+						JSONUtil.put(
+							"fields", JSONUtil.putAll(ddmFormField.getName())
```

Comment:
Do you think that it would be better to create a row for each field? 
Like: "rows": "[{"columns":[{"size":12,"fields":["Field1"]}]},{"columns":[{"size":12,"fields":["Field2"]}]}]",

Reply:
I'm doing that, I'm adding a row per field. See the **ddm-data-definition-json-converter-nested-fields-expected-result.json** Line 108:

```
"rows": "[{\"columns\":[{\"size\":12,\"fields\":[\"Text\"]}]},{\"columns\":[{\"size\":12,\"fields\":[\"Text\"]}]}]",
```
This is the expected result for conversion and we are checking in on tests.